ckanext/data_qld/test_plugin.py

```diff
@@ -22,41 +19,11 @@ def update_config(self, config):
 
     def get_actions(self):
         actions = (
-            qld_test_create_dataset,
-            qld_test_purge_dataset,
-            qld_test_create_resource_for_dataset,
             qld_test_trigger_notify_privacy_assessment_result,
         )
 
         return {"{}".format(func.__name__): func for func in actions}
 
-    # IXloader
-
-    def can_upload(self, resource_id):
-        context = _make_context()
-        res_dict = tk.get_action("resource_show")(context, {"id": resource_id})
-
-        return tk.asbool(res_dict.get("_xloader"))
-
-
-@tk.side_effect_free
-def qld_test_create_dataset(context, data_dict):
-    data_dict['resources'] = []
-    return DatasetFactory(**data_dict)
-
-
-@tk.side_effect_free
-def qld_test_purge_dataset(context, data_dict):
-    context = _make_context()
-    tk.get_action("dataset_purge")(context, data_dict)
-
-
-@tk.side_effect_free
-def qld_test_create_resource_for_dataset(context, data_dict):
-    data_dict['_xloader'] = bool(data_dict.get("xloader"))
-
-    return ResourceFactory(**data_dict)
-
 
 @tk.side_effect_free
 def qld_test_trigger_notify_privacy_assessment_result(context, data_dict):
```

Comment:
this could be done via the cli, no need for this test plugin then.

Reply:
True, I just haven't prioritised it yet as it hasn't interfered with running tests.